### ISSUE TITLE
pre-submission enquiry: invalid Git repository address

### ISSUE THREAD

**Issue by tliu68** [openjournals/joss#847]
Hello! 

We are preparing a submission to JOSS. Our software AutoGMM lives in a branch of a repository with this [link](https://github.com/microsoft/graspologic/tree/tom/autogmm_joss). But in the submission form, we have constantly seen an error that reads
> Invalid Git repository address. Check that the repository can be cloned using the value entered in the form, and that access doesn't require authentication.

Any idea about what could cause this error? Thanks in advance!

**Comment by arfon** [comment]
```
arfon@mawr ~ $ git ls-remote https://github.com/microsoft/graspologic/tree/tom/autogmm_joss
fatal: repository 'https://github.com/microsoft/graspologic/tree/tom/autogmm_joss/' not found

arfon@mawr ~ $ git ls-remote -q https://github.com/microsoft/graspologic 
81e32ea14ceaae1cb694a11baca522efe52efa22	HEAD
f6709b94325cb5a1aa745a8fdbdd9d14f5aae67b	refs/heads/add-pr-template
58ae3340e91284ed0166a4044400c58477f835c8	refs/heads/anomaly
f9752fd8858cc632b9c5e2d5be92f564ca6825d8	refs/heads/ase-alap
d2f8e43e501989ccb7cbd0354af7594f37af890a	refs/heads/ase-embed
9ad031d982ab6e1c0851e8cf447bcfff79b25567	refs/heads/build-validation-on-forked-prs
8b0d94bf3d00c6756cda806ba113a86799b5a73a	refs/heads/ci-publish
```

tl;dr - `https://github.com/microsoft/graspologic/tree/tom/autogmm_joss` is not a URL that we can clone, `https://github.com/microsoft/graspologic` is (please use https://github.com/microsoft/graspologic in the form).

I've struggled to phrase this in a way that doesn't cause confusion, and would welcome any suggestions on how to improve this!
